issuing_cardholder.go

```diff
@@ -38,17 +38,47 @@ type IssuingBillingParams struct {
 	Name    *string        `form:"name"`
 }
 
+// IssuingCardholderCompanyParams represents additional information about a
+// `business_entity` cardholder.
+type IssuingCardholderCompanyParams struct {
+	TaxID *string `form:"tax_id"`
+}
+
+// IssuingCardholderIndividualVerificationDocumentParams represents an
+// identifying document, either a passport or local ID card.
+type IssuingCardholderIndividualVerificationDocumentParams struct {
+	Back  *string `form:"back"`
+	Front *string `form:"front"`
+}
+
+// IssuingCardholderIndividualVerificationParams represents government-issued ID
+// document for this cardholder.
+type IssuingCardholderIndividualVerificationParams struct {
+	Document *IssuingCardholderIndividualVerificationDocumentParams `form:"document"`
+}
+
+// IssuingCardholderIndividualParams represents additional information about an
+// `individual` cardholder.
+type IssuingCardholderIndividualParams struct {
+	DOB          *DOBParams                                     `form:"dob"`
```

Comment:
You shouldn't re-use this. It's not shared in the spec so you need to duplicate this.